src/components/SelectTypeQuestion/SelectTypeQuestion.tsx

```diff
@@ -16,11 +16,22 @@ function SelectTypeQuestion() {
   let question;
   let type: `${QuestionTypes}`;
   let hasImages = false;
+  let instructions;
 
   if (state?.quiz.currentQuestion) {
     question = state.quiz.currentQuestion.next_question;
     type = question.type;
     hasImages = question.options.some((option: QuestionOption) => option.images);
+    switch (type) {
+      case QuestionTypes.MultipleSelect:
+        instructions = 'Select one or more options';
+        break;
+      case QuestionTypes.SingleSelect:
+        instructions = 'Select one option';
```

Comment:
Designs don't have `Select one option` if I'm looking at the correct thing. Do you mind checking in with Mariano to see if that's intentional? If it is, we can remove it from single select

Reply:
Sure thing, I'll drop him a ping

Reply:
According to Mariano, it was intentional as the original ux intent for single-select was to nav to the next question immediately when the user selects one option. I've removed the single select and made the code more concise.